tidy3d/components/simulation.py

```diff
@@ -549,7 +551,8 @@ def plot_structures_eps(
         hlim, vlim = Scene._get_plot_lims(
             bounds=self.simulation_bounds, x=x, y=y, z=z, hlim=hlim, vlim=vlim
         )
-
+        if freq is None and len(self.sources) > 0:
+            freq = self.sources[0].source_time.freq0
```

Comment:
Maybe keep it as None if there are multiple freq0. Like this:
https://github.com/flexcompute/tidy3d/blob/2f2f0814b415aca77b8ba76307d354a40265936e/tidy3d/components/medium.py#L179

Reply:
Agree with @caseyflex 's point. Rarelly plotting eps at infinite frequency would make physical sense or is what the user is looking for. Good to warn about it.